### DISCUSSION TITLE
windows版新安装之后闪退

### DISCUSSION THREAD

**Discussion by konaisland** [getlantern/lantern-forum-cn#666] [General]
使用的版本就是现在github上下载的stable版，联通线路

**Comment by MaxMKMax** [comment]
@konaisland 您好，谢谢您的反馈。蓝灯已经在正视这个问题了。 我们正在修复问题和进行优化。请您耐心等待。非常抱歉影响了您的体验感。谢谢您的等待与支持。

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Reply by MaxMKMax** [reply]
@konaisland 您好. 请您提供我报告票号. 有了票号我们可以更容易找到问题然后解决. 谢谢.

**Comment by XCLlantern** [comment]
我也遇到了这个问题，换成测试版以后可以用，但是有一个新的问题，就是无法获得授权转成正式版。哪怕另一台电脑上输入了授权码显示授权成功，也迟迟无法转成正式版。希望尽快解决

**Reply by MaxMKMax** [reply]
@XCLlantern 您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

**Reply by XCLlantern** [reply]
好的，谢谢！那我明天在安装了专业版的电脑上提交log。这台电脑目前的版本是测试版（lantern-installer-preview）还是免费版，没找到提交log的地方

**Reply by XCLlantern** [reply]
刚刚在这个测试版的蓝灯应用程序上找到了"report issue"的入口，但是无法成功提交。点击提交以后，弹出对话框“report_issue_error”.

**Reply by XCLlantern** [reply]
如果方便，能否帮我查一下我的激活码？我的邮箱是fansizhe2013@sina.com。我的专业版目前截止日期是2027年某月某日，好像续费是去年11月左右。我可以试试能否通过激活码把我的测试版转成专业版。

**Reply by Ruin-cs** [reply]
我一样的问题，刚试了下载那个供win7用的是7.8.2版本的，重新授权一次可以用了，但神奇的是现在显示之前在测试版授权的也占了一个授权位置，就是我一台电脑占俩名额，还无法移除……

**Reply by XCLlantern** [reply]
我刚刚撤退了，安装了windows-7的稳定版，马上通过邮件获得授权码转成正式版了。

**Comment by MaxMKMax** [comment]
@XCLlantern @Ruin-cs @konaisland 
您好. 非常抱歉影响了您的体验感. 您那里能帮我提交报告然后给我 8 票号.吗? 如果应用闪退, 提交不了, 您可以手动帮我找log然后提供给我吗? 非常感谢! 有log我们就能马上修复了. 麻烦您帮忙了. 谢谢! 
可以是看这么找:
C:\Users\username\AppData\Roaming\Lantern

各位放心. 我已经把你们的问题提交给技术团队了. 我们会尽快修复的. 谢谢您的耐心等待. 也很抱歉这几天影响了你们的体验感. 谢谢你们的反馈也感谢你们支持蓝灯!

**Comment by helanhao** [comment]
闪退的解决办法是：安装win7版本的就可以了，安装之后它会自动升级到7.8.4版本，不会闪退了

**Comment by MaxMKMax** [comment]
@konaisland 各位好. 可以帮我们提交一个应用内报告吗? 或者用手动方法给我们log. 谢谢. 明天开会我帮你们反馈问题给技术团队, 尽快修复. 麻烦你们啦. 谢谢帮忙.

**Comment by ycjdixon** [comment]
7.8.6版本在首次使用过一段时间（几分钟到几小时不等）后会自动退出。只要一次退出过后，再次打开就一直闪退了。
  票号：74657423  @Mike777Khoo 我特意安装了7.8.6版本后提交的，还记得你之前答应过我的事情吗？

**Reply by MaxMKMax** [reply]
@ycjdixon 好的. 今晚 11 点我会和技术团队他们开会. 尽我的能力帮你们解决问题. 谢谢您的Log.

**Comment by ycjdixon** [comment]
[Uploading lantern.log…]()
 @Mike777Khoo 闪退后的log 上传给你了

**Comment by ycjdixon** [comment]
@Mike777Khoo 新版本修复好了吗？下周7.8.5版本很难连上网了。

**Comment by ycjdixon** [comment]
@Mike777Khoo 新版本修复好了吗？现在7.8.5版本很难连上网了。

**Reply by MaxMKMax** [reply]
@ycjdixon 您可以下载到更新版本吗? 您这个版本好像不是最新的.
https://github.com/getlantern/lantern
7.8.8

**Reply by ycjdixon** [reply]
@Mike777Khoo这个连接下载安装的还是7.8.6版本，依然闪退。
